classes/utils/__init__.py

```diff
@@ -4,3 +4,4 @@
 from . import Thread
 from . import Utils
 from . import Vector
+from . import Config
```

Comment:
I dislike the red circle for absolutely no reason at all

Reply:
I don't understand what you mean

Reply:
No newline at the end of file